### PR TITLE
2024-03-28-meddra_hlt_pt_mapper_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

**Review by muhammetsnts — APPROVED** [review]

File: docs/_posts/akrztrk/2024-03-28-ner_profiling_deidentification_en.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pipeline is designed for deidentification in clinical texts, leveraging a range of pretrained NER models tailored for extracting and anonymizing sensitive information. By integrating these models, the pipeline provides a comprehensive solution for protecting patient privacy and complying with data protection regulations.
+
+The pipeline employs `embeddings_clinical` for contextual understanding and includes the following specialized NER models for deidentification:
+
+`ner_deid_augmented`, `ner_deid_enriched`, `ner_deid_generic_augmented`, `ner_deid_name_multilingual_clinical`, `ner_deid_sd`, `ner_deid_subentity_augmented`, `ner_deid_subentity_augmented_i2b2`, `ner_deid_synthetic`, `ner_jsl`, 'ner_jsl_enriched'
```
Comment (muhammetsnts) [inline docs/_posts/akrztrk/2024-03-28-ner_profiling_deidentification_en.md:25]: Fix ner_jsl_enriched quotes.

**Review by muhammetsnts — CHANGES_REQUESTED** [review]
@RakshitKhajuria

File: docs/_posts/akrztrk/2024-03-28-ner_profiling_deidentification_en.md (hunk 1 above)
Comment (RakshitKhajuria) [reply docs/_posts/akrztrk/2024-03-28-ner_profiling_deidentification_en.md]: Fixed @muhammetsnts

File: docs/_posts/akrztrk/2024-03-28-ner_profiling_deidentification_en.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+|Type:|pipeline|
+|Compatibility:|Healthcare NLP 5.3.1+|
+|License:|Licensed|
+|Edition:|Official|
+|Language:|en|
+|Size:|2.0 GB|
+
+## Included Models
+
+- DocumentAssembler
```
[the remark this answers is not in the document]
Comment (RakshitKhajuria) [reply docs/_posts/akrztrk/2024-03-28-ner_profiling_deidentification_en.md:155]: Done !

**Pull request merged**